diff --git a/node_modules/@react-native-community/cli-platform-ios/build/commands/runIOS/index.js b/node_modules/@react-native-community/cli-platform-ios/build/commands/runIOS/index.js
--- a/node_modules/@react-native-community/cli-platform-ios/build/commands/runIOS/index.js
+++ b/node_modules/@react-native-community/cli-platform-ios/build/commands/runIOS/index.js
@@ -219,7 +219,13 @@
   const activeDeveloperDir = _child_process().default.execFileSync('xcode-select', ['-p'], {
     encoding: 'utf8'
   }).trim();
-  _child_process().default.execFileSync('open', [`${activeDeveloperDir}/Applications/Simulator.app`, '--args', '-CurrentDeviceUDID', selectedSimulator.udid]);
+  const simulatorApp = _path().default.join(activeDeveloperDir, 'Applications', 'Simulator.app');
+  const deviceHubApp = _path().default.join(activeDeveloperDir, '..', 'Applications', 'DeviceHub.app');
+  if (_fs().default.existsSync(simulatorApp)) {
+    _child_process().default.execFileSync('open', [simulatorApp, '--args', '-CurrentDeviceUDID', selectedSimulator.udid]);
+  } else if (_fs().default.existsSync(deviceHubApp)) {
+    _child_process().default.execFileSync('open', [`devices://device/open?id=${selectedSimulator.udid}`]);
+  }
   if (selectedSimulator.state !== 'Booted') {
     bootSimulator(selectedSimulator);
   }
